src/main.py

```
@@ -642,11 +915,32 @@ def main():
     application = Application.builder().token(bot_token).build()
     application.add_handler(MessageHandler(filters.TEXT & ~filters.COMMAND, handle_message))
     # This handler will receive every error which happens in your bot
     application.add_error_handler(error_handler)
+
+    # Start cleanup task after event loop is running
+    async def post_init(app):  # pylint: disable=unused-argument
+        global cleanup_task  # pylint: disable=global-statement
+        cleanup_task = asyncio.create_task(cleanup_stale_users())
+
+    # Cancel cleanup task on shutdown
+    async def post_shutdown(app):  # pylint: disable=unused-argument
```

Ruff (0.15.2)

[warning] Unused function argument: `app`

(ARG001)